nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -132,7 +138,9 @@ else
     system_root=$closure
     # Create a temporary file ending in .closure (so nixos-prepare-root knows to --import it) to transport the store closure
     # to the filesytem we're preparing. Also delete it on exit!
```

Comment:
I'd say it still does, it's a FIFO rather than a regular file but I think we're still deleting it on exit?